app/translations.js

```diff
@@ -220,43 +220,49 @@ const translations = {
   'sv': {
     'frontpage': 'Framsidan',
     'stop-page.title-short': 'Hållplats',
+    'stop-page.title': 'Hållplats {stop_name} - {stop_code}',
```

Comment:
It seems the translation is based on an older file, as the variable names, were changed in https://github.com/HSLdevcom/digitransit-ui/pull/1212/files